### PR TITLE
New pr

### PR DESCRIPTION
**Pull request by citrus3301**
I made the new md files for sending payloads, this includes only adding a new section, and not changing the old one, and using the website you reccomended. Hopefully its better, but if anything needs to be changed, let me know!

### CODE REVIEW

File: docs/user_guide/sysnand/sending_payload.md [hunk 1]
```diff
@@ -79,6 +79,26 @@ Now that the device is in RCM, we will need to send it a payload. The methods ar
...
+
+!!! tip ""
+    - The latest release of <a href="https://github.com/CTCaer/hekate/releases/" target="_blank">Hekate</a> (either the hekate_ctcaer bin or the hekate_ctcaer zip)
+    - A USB-C cable
+    - If your chromebook has a USB-C note that this will not work using a C-C cable
+	
+### Instructions
+    
+!!! tip ""
+    1. Go to the website https://switchgui.de/web-payload/ and scroll all the way down
```
Comment (suchmememanyskill) [inline docs/user_guide/sysnand/sending_payload.md]: You are aware you can use `[this](https://switchgui.de/web-payload/)` to show links, right? like [this](https://switchgui.de/web-payload/)

**Comment by suchmememanyskill** [comment]
Also, it seems like you copied the sysmmc guide's sending_payload.md to the emummc guide's sending_payload.md. This removes tegraexplorer and some other important edits, as seen in your changes to the file. please revert

**Comment by noirscape** [comment]
The point isn't to nuke tegraexplorer entirely from all the instructions...

**Comment by citrus3301** [comment]
@suchmememanyskill I have made the link more user-friendly, and I have removed the tegraexplorer from the sysnand version, any other changes i should make?

**Comment by citrus3301** [comment]
I see about the tegraexplorer and misunderstood at first, I'll fix it asap and get it working

**Comment by suchmememanyskill** [comment]
Closing for pr #100 (has your instructions, but isn't a mess with the files)

**Pull request closed without merging**